x/earn/types/genesis.go

```diff
@@ -38,3 +44,30 @@ func DefaultGenesisState() GenesisState {
 		VaultShareRecords{},
 	)
 }
+
+func Kava11GenesisState(accountKeeper AccountKeeper) GenesisState {
```

Comment:
should this be moved to the migration PR?

Reply:
ah right oops, seem to have accidentally included it in a different commit some time ago